.github/workflows/build.yml

```diff
@@ -0,0 +1,153 @@
+name: Kirby CI
+on:
+  push:
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  lint_and_unittest:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: '10.16'
+      - name: run npm ci
+        run: npm ci
+      - name: run npm lint
+        run: npm run lint
+      - name: run npm extract-scss-variables
+        run: npm run extract-scss-variables
+      - name: run npm test ci
+        run: npm run test:ci
+      - name: run npm coverage-report
+        run: npm run coverage-report || true
+
+  build_kirby:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: '10.16'
+      - name: run npm ci
+        run: npm ci
+      - name: run npm dist:designsystem
+        run: npm run dist:designsystem
+
+  bump_version:
+    needs: [lint_and_unittest]
+    if: "!startsWith(github.event.head_commit.message, 'Bumping version to') && github.ref == 'refs/heads/master'"
```

Comment:
Consider having the commit message in a variable, so it can be used both bumping/tagging + when checking whether to run this job...